pom.xml

```diff
@@ -181,7 +178,6 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-project-info-reports-plugin</artifactId>
-          <version>${maven-project-info-reports-plugin.version}</version>
```

Comment:
Why don't we set any version for these plugins?
This way we will use the version that is coming from apache parent right?